### PR TITLE
Create Factorial_without_recursion.cpp

### LINKED ISSUE #57: Add Factorial function without recursion using C++
I would like to work on this issue .Please assign it o me.

### PR DESCRIPTION
**Pull request by Subhangini**
Fixes #57

### CODE REVIEW

**Comment by Subhangini** [comment]
@nakul- 19 and @rudrakshi99  please review it

File: Code/C++/Factorial_without_recursion.cpp [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    for(int i=num; i>1; i--) 
+    {
+        fact=fact*i;
+    }
+    cout<<"Factorial of "<<num<<" is "<<fact;
+    return 0;
+}
+/*
+Test case :
+	Input : 5
```
Comment (rudrakshi99) [inline Code/C++/Factorial_without_recursion.cpp:20]: add atleast 2 test cases

**Review by rudrakshi99 — CHANGES_REQUESTED** [review]

**Comment by rudrakshi99** [comment]
update the readme in the Number theory folder.

**Comment by Subhangini** [comment]
ok

**Comment by Subhangini** [comment]
made the changes .Review it

File: Code/C++/Factorial_without_recursion.cpp [hunk 2]
```diff
@@ -16,9 +16,15 @@ int main()
...
+Test case 1:
 	Input : 5
 	Output : 120
-	Time complexity: O(n)
+Test case 1:
+	Input : 3
+	Output : 6
+	
+*/
+/*
```
Comment (rudrakshi99) [inline Code/C++/Factorial_without_recursion.cpp:27]: remove this opening and closing comment from here.

**Review by rudrakshi99 — CHANGES_REQUESTED** [review]

**Comment by Subhangini** [comment]
done @rudrakshi99. sorry for the delay. Review the changes

**Pull request merged**